sdk-exporter/prometheus/src/main/scala/org/typelevel/otel4s/sdk/exporter/prometheus/autoconfigure/PrometheusMetricExporterAutoConfigure.scala

```diff
@@ -40,9 +42,9 @@ import org.typelevel.otel4s.sdk.metrics.exporter.MetricExporter
   * | otel.exporter.prometheus.port                    | OTEL_EXPORTER_PROMETHEUS_PORT                    | The port that metrics are served on. Default is `9464`.                                  |
   * | otel.exporter.prometheus.default.aggregation     | OTEL_EXPORTER_PROMETHEUS_DEFAULT_AGGREGATION     | Default aggregation as a function of instrument kind. Default is `default`.              |
   * | otel.exporter.prometheus.without.units           | OTEL_EXPORTER_PROMETHEUS_WITHOUT_UNITS           | If metrics are produced without a unit suffix. Default is `false`.                       |
-  * | otel.exporter.prometheus.without.type.suffixes   | OTEL_EXPORTER_PROMETHEUS_WITHOUT_TYPE_SUFFIXES   | If metrics are produced without a type suffix. Default is `false`.                       |
-  * | otel.exporter.prometheus.disable.scope.info      | OTEL_EXPORTER_PROMETHEUS_DISABLE_SCOPE_INFO      | If metrics are produced without a scope info metric or scope labels. Default is `false`. |
-  * | otel.exporter.prometheus.disable.target.info     | OTEL_EXPORTER_PROMETHEUS_DISABLE_TARGET_INFO     | If metrics are produced without a target info metric. Default is `false`.                |
+  * | otel.exporter.prometheus.without.type.suffix     | OTEL_EXPORTER_PROMETHEUS_WITHOUT_TYPE_SUFFIX     | If metrics are produced without a type suffix. Default is `false`.                       |
+  * | otel.exporter.prometheus.without.scope.info      | OTEL_EXPORTER_PROMETHEUS_WITHOUT_SCOPE_INFO      | If metrics are produced without a scope info metric or scope labels. Default is `false`. |
+  * | otel.exporter.prometheus.without.target.info     | OTEL_EXPORTER_PROMETHEUS_WITHOUT_TARGET_INFO     | If metrics are produced without a target info metric. Default is `false`.                |
```

Comment:
That's what I found:
https://github.com/open-telemetry/opentelemetry-configuration/blob/8efa0ae7cb9bdee774dcb3efd8545fc534507d75/schema/meter_provider.json#L116-L137